### PR TITLE
Page layout patches

### PR DESCRIPTION
**Pull request by jasongao97**
Modified according to #672

- Acknowledgement / Introduction start on right-side page
- Add page numbers to the front matter pages
- Remove page numbers on blank pages

### CODE REVIEW

**Comment by jasongao97** [comment]
Added CSS [properties](https://www.princexml.com/doc/paged/#widows-and-orphans) to avoid orphans or widows

```css
p {
  orphans: 2;
  widows: 2;
}
```

Ref #665

File: magicbook/stylesheets/components/typography.scss [hunk 1]
```diff
@@ -12,6 +12,8 @@ p {
   margin-top: 10pt;
   margin-bottom: 10pt;
   font-weight: $font-book;
+  orphans: 2;
```
Comment (shiffman) [inline magicbook/stylesheets/components/typography.scss]: @jasongao97 I noticed you added these, do we know how well they work? Should we be investigating "prince for books" as well?

Comment (jasongao97) [reply magicbook/stylesheets/components/typography.scss]: > @jasongao97 I noticed you added these, do we know how well they work? Should we be investigating "prince for books" as well?

Yes! They effectively prevent orphan lines at the bottom or top of a page. Prince book is for better line-break and spread-balancing which I will investigate today.

**Comment by jasongao97** [comment]
Duplication of #746

**Pull request closed without merging**